### PR TITLE
[FLOC-2151] Add warning about restarting VMs to tutorial doc

### PR DESCRIPTION
**Pull request by dwgebler**
Fixes FLOC-2151 via documenting the workaround for loss of static IP config,

### CODE REVIEW

**Comment by Azulinho** [comment]
The buildbot tests are failling, but from what I can see this is because they are validating the 'live' website and not the latest-dev

http://build.clusterhq.com/builders/flocker-docs/builds/8032/steps/build-linkcheck/logs/errors

this if I am reading these correctly.
@dwgebler Can you comment, merge and close if that's the case?

**Comment by adamtheturtle** [comment]
Why is this in the tutorial and not the GSG?

File: docs/using/tutorial/vagrant-setup.rst [hunk 1]
```diff
@@ -87,6 +87,11 @@ One important thing to note is that these VMs are statically assigned the IPs ``
 These two IP addresses will be used throughout the tutorial and configuration files.
 
 .. warning::
+
+   On some versions of Vagrant and Virtualbox, restarting the tutorial virtual machines via the ``vagrant halt`` and ``vagrant up`` commands can result in losing the static IP configuration, making the nodes unreachable on the assigned ``172.15.255.25x`` addresses.
```
Comment (adamtheturtle) [inline docs/using/tutorial/vagrant-setup.rst]: Virtualbox > VirtualBox

File: docs/using/tutorial/vagrant-setup.rst [hunk 2]
```diff
@@ -87,6 +87,11 @@ One important thing to note is that these VMs are statically assigned the IPs ``
...
+   In this case you should destroy and recreate the machines with the ``vagrant destroy`` and ``vagrant up`` commands.
+
+.. warning::
```
Comment (adamtheturtle) [inline docs/using/tutorial/vagrant-setup.rst]: This warning is now out of context - it follows from the sentence above it.

**Comment by adamtheturtle** [comment]
Sorry, because of the points I've just added (particularly the GSG) can this please be resubmitted.

**Comment by Azulinho** [comment]
the GSG is going away, as per PR https://github.com/ClusterHQ/flocker/pull/1506

**Comment by dwgebler** [comment]
> Why is this in the tutorial and not the GSG?

I'm not doing anything to the GSG in this branch because it's being removed in another branch.

**Comment by dwgebler** [comment]
> Can you comment, merge and close if that's the case?

Broken link warnings are pretty normal, there are none in here that are "genuine" as it were, I think most of them happen because we can't retrieve HTTPS on that builder for some reason. In the first build there was also a spelling error but I've fixed that now. @Azulinho if you'd like me to merge this, please accept the JIRA issue https://clusterhq.atlassian.net/browse/FLOC-2151 thanks :)

**Comment by adamtheturtle** [comment]
@dwgebler Please can you make a follow-up to make any relevant issues for the new guide.

**Comment by adamtheturtle** [comment]
:disappointed: The GSG is here to stay for 1.0.0.

This needs to be merged into release/flocker-1.0.0pre1 too.

**Pull request merged**